packages/use-contractkit/src/walletIcons.tsx

```diff
@@ -523,3 +523,32 @@ export const CELO_DANCE: React.FC<React.SVGProps<SVGSVGElement>> = (props) => (
     </g>
   </svg>
 );
+
+export const CELO_TERMINAL: React.FC<React.SVGProps<SVGSVGElement>> = (
```

Comment:
Ive come to the determination that unless you are modifying your svg with js. your best off just treating it like png or other image file. otherwise you have larger javascript files from all that vector data.